test/service/test_jobs.py

```diff
@@ -130,47 +130,47 @@ def test_assigns_correct_job_id(self):
         self.assertEqual('test-new-job-id', job.job_id)
 
     def test_assigns_correct_name(self):
-        self.mock_get_algo.return_value = create_algorithm()
+        self.mock_get_algo.return_value = create_algorithm('pzsvc-ndwi-py')
```

Comment:
`pzsvc-ndwi-py` is the default value for the `create_algorithm` function.  You don't need to repeat it here.